Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -26,6 +26,7 @@ report_panel = Ext.create('Ext.panel.Panel', {
     }, {
         xtype: 'panel',
         ref: 'graph_reports',
+        graphs_added: true,
```

Comment:
right,
when we have them in `false` user can click other nodes but those reports will not start loading. (see lines 197-198)